doc/api/n-api.md

```diff
@@ -2,7 +2,7 @@
 
 <!--introduced_in=v7.10.0-->
 
-> Stability: 1 - Experimental
+> Stability: 2 - Stable
```

Comment:
Do we track changes like this in the documentation with YAML metadata?

Reply:
Looking at the existing docs, I don't think so.